<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ú–µ–Ω–µ–¥–∂–µ—Ä –ª–∏—á–Ω—ã—Ö —Ñ–∏–Ω–∞–Ω—Å–æ–≤</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #2E3440;
    color: #ECEFF4;
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
}

QMainWindow {
    background-color: #2E3440;
}

QPushButton {
    background-color: #4C566A;
    color: #ECEFF4;
    border: 1px solid #4C566A;
    border-radius: 8px;
    padding: 10px;
    font-size: 11pt;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #5E81AC;
}

QPushButton:pressed {
    background-color: #81A1C1;
}

/* Accent button style */
QPushButton#addTransactionButton, QPushButton#pushButton_saveTransaction {
    background-color: #88C0D0;
    color: #2E3440;
}

QPushButton#addTransactionButton:hover,
QPushButton#pushButton_saveTransaction:hover {
    background-color: #8FBCBB;
}

QPushButton#addTransactionButton:pressed,
QPushButton#pushButton_saveTransaction:pressed {
    background-color: #A3BE8C;
}

/* Destructive action button style */
QPushButton#deleteTransactionButton, QPushButton#stopServerButton {
    background-color: #BF616A;
    color: #ECEFF4;
}

QPushButton#deleteTransactionButton:hover,
QPushButton#stopServerButton:hover {
    background-color: #D08770;
}

/* New styles for showAllButton and clearHistoryButton */
QPushButton#showAllButton {
    background-color: #88C0D0;
    color: #2E3440;
}

QPushButton#showAllButton:hover {
    background-color: #8FBCBB;
}

QPushButton#showAllButton:pressed {
    background-color: #A3BE8C;
}

QPushButton#clearHistoryButton {
    background-color: #BF616A;
    color: #ECEFF4;
}

QPushButton#clearHistoryButton:hover {
    background-color: #D08770;
}

QLineEdit, QTextEdit, QSpinBox {
    background-color: #3B4252;
    border: 1px solid #4C566A;
    border-radius: 5px;
    padding: 8px;
    font-size: 10pt;
    color: #ECEFF4;
}

QLineEdit:focus, QTextEdit:focus, QSpinBox:focus {
    border-color: #88C0D0;
}

QLabel {
    background-color: transparent;
    color: #ECEFF4;
    font-size: 10pt;
}

QLabel#label_title, QLabel#label_addTitle, QLabel#label_transactionsTitle, QLabel#label_historyTitle {
    font-size: 24pt;
    font-weight: bold;
    color: #ECEFF4;
}

QTableWidget {
    background-color: #3B4252;
    border: 1px solid #4C566A;
    gridline-color: #4C566A;
    color: #D8DEE9;
    font-size: 10pt;
}

QTableWidget::item {
    padding: 5px;
}

QHeaderView::section {
    background-color: #434C5E;
    color: #ECEFF4;
    padding: 6px;
    border: none;
    font-weight: bold;
    font-size: 11pt;
}

QTableWidget::item:selected {
    background-color: #88C0D0;
    color: #2E3440;
}

QRadioButton {
    font-size: 11pt;
    spacing: 10px;
}

QRadioButton::indicator {
    width: 15px;
    height: 15px;
}

QRadioButton::indicator::unchecked {
    border: 1px solid #4C566A;
    background-color: #3B4252;
    border-radius: 7px;
}

QRadioButton::indicator::checked {
    border: 1px solid #88C0D0;
    background-color: #88C0D0;
    border-radius: 7px;
}

QStackedWidget {
    background-color: #2E3440;
}

QMenuBar {
    background-color: #3B4252;
    color: #ECEFF4;
}

QMenuBar::item:selected {
    background-color: #4C566A;
}

QStatusBar {
    background-color: #3B4252;
    color: #ECEFF4;
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 20px;
    height: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageMainMenu">
       <layout class="QVBoxLayout" name="verticalLayout_main">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <spacer name="spacerTop">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_title">
          <property name="text">
           <string>–ì–ª–∞–≤–Ω–æ–µ –º–µ–Ω—é</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="buttonsLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="launchServerButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üöÄ –ó–∞–ø—É—Å—Ç–∏—Ç—å —Å–µ—Ä–≤–µ—Ä</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="stopServerButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üõë –û—Å—Ç–∞–Ω–æ–≤–∏—Ç—å —Å–µ—Ä–≤–µ—Ä</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="addTransactionButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>‚ûï –î–æ–±–∞–≤–∏—Ç—å —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—é</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="editTransactionButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üñãÔ∏è –†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—é</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="deleteTransactionButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üóëÔ∏è –£–¥–∞–ª–∏—Ç—å —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—é</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="showAllButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üßæ –ü–æ–∫–∞–∑–∞—Ç—å –≤—Å–µ —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏–∏</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="showHistoryButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üìú –ò—Å—Ç–æ—Ä–∏—è –¥–µ–π—Å—Ç–≤–∏–π</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="clearHistoryButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üßπ –û—á–∏—Å—Ç–∏—Ç—å –∏—Å—Ç–æ—Ä–∏—é</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="exitButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>üö™ –í—ã–π—Ç–∏</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacerBottom">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAddTransaction">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>100</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>100</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_addTitle">
          <property name="text">
           <string>–ù–æ–≤–∞—è —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—è</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_add">
          <property name="labelAlignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
          </property>
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>–¢–∏–ø:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioButton_income">
              <property name="text">
               <string>–î–æ—Ö–æ–¥</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_expense">
              <property name="text">
               <string>–†–∞—Å—Ö–æ–¥</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>–ö–∞—Ç–µ–≥–æ—Ä–∏—è:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_category">
            <property name="placeholderText">
             <string>–ù–∞–ø—Ä–∏–º–µ—Ä, –ø—Ä–æ–¥—É–∫—Ç—ã</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>–î–∞—Ç–∞:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="spinBox_day">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>31</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_month">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_year">
              <property name="minimum">
               <number>2000</number>
              </property>
              <property name="maximum">
               <number>2100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>–°—É–º–º–∞:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_amount">
            <property name="placeholderText">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>–û–ø–∏—Å–∞–Ω–∏–µ:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QTextEdit" name="textEdit_description">
            <property name="placeholderText">
             <string>–ù–∞–ø—Ä–∏–º–µ—Ä, –∑–∞—Ä–ø–ª–∞—Ç–∞ –∑–∞ –º–∞–π</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_addStatusMessage">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_cancelAddTransaction">
            <property name="text">
             <string>‚¨ÖÔ∏è –ù–∞–∑–∞–¥</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_saveTransaction">
            <property name="text">
             <string>üíæ –°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageTransactionsView">
       <layout class="QVBoxLayout" name="verticalLayout_view">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_transactionsTitle">
          <property name="text">
           <string>–°–ø–∏—Å–æ–∫ —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏–π</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_transactions">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_backToMenu">
          <property name="text">
           <string>‚¨ÖÔ∏è –ù–∞–∑–∞–¥ –≤ –º–µ–Ω—é</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageHistory">
       <layout class="QVBoxLayout" name="verticalLayout_history">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_historyTitle">
          <property name="text">
           <string>–ò—Å—Ç–æ—Ä–∏—è –¥–µ–π—Å—Ç–≤–∏–π</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_history">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_backFromHistory">
          <property name="text">
           <string>‚¨ÖÔ∏è –ù–∞–∑–∞–¥ –≤ –º–µ–Ω—é</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
